[dbt] More small changes (#4574)
ClickHouse/clickhouse-docs: docs/integrations/data-ingestion/etl-tools/dbt/features-and-configurations.md

| ---------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
| engine                 | The table engine (type of table) to use when creating tables                                                                                                                                                                                                                                                         | `MergeTree()`  |
| order_by               | A tuple of column names or arbitrary expressions. This allows you to create a small sparse index that helps find data faster.                                                                                                                                                                                        | `tuple()`      |

> | partition_by           | A partition is a logical combination of records in a table by a specified criterion. The partition key can be any expression from the table columns.                                                                                                                                                                 |                |

Reviewer: ```suggestion
| partition_by           | A custom [partition key](/engines/table-engines/mergetree-family/custom-partitioning-key). In most cases, specifying a partition key is **not recommended**.                                                                                                                                                                 |                |
```
